src/sentry/static/sentry/app/views/releases/releaseCommits.jsx

```diff
@@ -83,7 +81,9 @@ const ReleaseCommits = React.createClass({
     return {
       loading: true,
       error: false,
-      commitList: []
+      commitList: [],
+      activeRepo: null,
+      title: 'All Repositories'
```

Comment:
Oh oops that was for when I was making the dropdown not just a caret. Will remove.